modules/sdk/src/main/scala/org/tessellation/sdk/cli/snapshot.scala

```diff
@@ -27,14 +27,18 @@ object snapshot {
     .env[Path]("CL_INCREMENTAL_SNAPSHOT_TMP_STORED_PATH", help = "Path to tmp storage of incremental snapshot")
     .withDefault(Path("data/incremental_snapshot_tmp"))
 
-  val opts = (snapshotPath, incrementalPersistedSnapshotPath, incrementalTmpSnapshotPath).tupled.mapValidated {
-    case (snapshotPath, incrementalPersistedSnapshotPath, incrementalTmpSnapshotPath)
+  val snapshotInfoPath: Opts[Path] = Opts
+    .env[Path]("CL_SNAPSHOT_INFO_PATH", help = "Path to store snapshot infos")
+    .withDefault(Path("data/snapshot_info"))
+
+  val opts = (snapshotPath, incrementalPersistedSnapshotPath, incrementalTmpSnapshotPath, snapshotInfoPath).tupled.mapValidated {
+    case (snapshotPath, incrementalPersistedSnapshotPath, incrementalTmpSnapshotPath, snapshotInfoPath)
         if snapshotPath =!= incrementalPersistedSnapshotPath && incrementalPersistedSnapshotPath =!= incrementalTmpSnapshotPath =>
```

Comment:
You didn't add check for snapshot info directory. I think we could take all the paths (string values?), put them in a Set and check if the size is as expected to be sure all paths are different.

Reply:
Good idea :+1: I'll add that